html/apps/oneClickDeploy/app.js

```diff
@@ -0,0 +1,6 @@
+var app = angular.module("PeersCount", []);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2620055748/issues/source?bid=3288751&fileBranchId=3313965#l1)